src/SignalR/clients/java/signalr/signalr.javaproj

```diff
@@ -2,7 +2,6 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
 
   <PropertyGroup>
-    <VersionPrefix>1.0.$(AspNetCorePatchVersion)</VersionPrefix>
```

Comment:
Is this being set somewhere else?

Reply:
https://github.com/aspnet/AspNetCore/blob/3c09d644cccdb21801f7a79e1188a1a1212de5d9/version.props#L44

Reply:
Ah got it. We had to override because the SignalR version didn't match the rest of the product.